core/src/main/java/org/apache/calcite/runtime/CompressionFunctions.java

```diff
@@ -62,4 +65,21 @@ public static ByteString compress(String data) {
     }
   }
 
+  /**
+   * MySql Decompression is based on zlib.
+   * <a href="https://docs.oracle.com/javase/8/docs/api/java/util/zip/Inflater.html">Inflater</a>
+   * is used to implement decompression.
+   */
+  public static String uncompress(ByteString data) {
+    try {
+      if (data == null || data.length() == 0) {
```

Comment:
For `data.length==0`, is it possible to be an empty string?

Reply:
> For `data.length==0`, is it possible to be an empty string?

```
mysql> select uncompress(null),uncompress('');
+------------------+----------------+
| uncompress(null) | uncompress('') |
+------------------+----------------+
| NULL             |                |
+------------------+----------------+
1 row in set (0.00 sec)
```
It should be an empty string

Reply:
Added new test case. Thanks :)